lib/RGBot.js

```diff
@@ -421,6 +421,37 @@ class RGBot extends EventEmitter {
         return null
     }
 
+    /**
+     * Gets the team name for this bot
+     * @return {string|null} Name of the team that this bot is on
+     */
+    myTeam() {
+        return this.teamForPlayer(this.username())
+    }
+
+    /**
+     * Returns a list of all players on the same team as this bot.
+     * @param {boolean} includeSelf if true, includes this bot in the list
+     * @return {string[]} A list of all player usernames on the same team as this bot.
+     */
+    getTeammateUsernames(includeSelf) {
```

Comment:
@abeizer can you elaborate on this? Do you mean a function that returns something like an object that has the team name and the team player objects? Happy to add that as well, but I was hoping to keep this one because this seemed like the most common scenario (since a lot of our functions and objects use the usernames as the main point of reference)

Reply:
And we're getting really close to just returning what the match info model presents in the first place :)